fvm/transactionVerifier.go

```diff
@@ -217,3 +222,25 @@ func (v *TransactionSignatureVerifier) sigIsForProposalKey(
 ) bool {
 	return txSig.Address == proposalKey.Address && txSig.KeyIndex == proposalKey.KeyIndex
 }
+
+func (v *TransactionSignatureVerifier) checkSignatureDuplications(tx *flow.TransactionBody) error {
+	observedSigs := make(map[string]bool)
+	for _, sig := range tx.PayloadSignatures {
```

Comment:
What is the reason of looking for duplicates in `PayloadSignatures` and `EnvelopeSignatures` separately, while access node code looks for duplicates in both sets together?

Reply:
I did separate them since we emit different errors in fvm side and might have different fee costs for users.

Reply:
Ah my bad, I didn't notice that the code is re-using the same map in the second loop. All good then